### DISCUSSION TITLE
github-linguist command shows the correct statistics but GitHub server is not getting updated after a few days.

### DISCUSSION THREAD

**Discussion by FeriBolour** [github-linguist/linguist#6230] [Q&A] [answered]
Hello Everyone,

I've been trying to work with github-linguist for a while and it still gives a bit of problem.

After a few days of trying to figure out different issues with the language statistics, I managed to set .githubattributes with the right commands to get the correct statistics from github-linguist command:
![image](https://user-images.githubusercontent.com/57817579/209871341-e92b10dc-fbfc-4043-8b6a-574a67957cba.png)

But in GitHub's webpage/server it's still not showing up:
![image](https://user-images.githubusercontent.com/57817579/209871508-b6d27b95-2f7d-4cf6-ae6c-2d4f4c480d51.png)

This is the link to the repo I am working on: [https://github.com/FeriBolour/cotton-phenotyping](https://github.com/FeriBolour/cotton-phenotyping)

I tried different solutions:

1. Waited almost a week.
2. Pushed meaningless stuff to the repo to update the server.

It's still not working. What do you suggest me t do.

**Comment by lildude** [comment, marked as answer]
You don't appear to have pushed your local changes as my analysis of a fresh clone of your repo shows:

```console
$ github-linguist cotton-phenotyping/
92.43%  2666651    Jupyter Notebook
5.32%   153376     Python
2.25%   64884      MATLAB
$
```

... which matches what GitHub is reporting.

**Reply by lildude** [reply]
Further to this, if you add `--breakdown` to the command we see these Jupyter files are still being considered:

```
Jupyter Notebook:
Model-Development/.ipynb_checkpoints/MaskRCNN-checkpoint.ipynb
Model-Development/.ipynb_checkpoints/MaskRCNN_Detectron2_NewBaselines-checkpoint.ipynb
Model-Development/.ipynb_checkpoints/MaskRCNN_wCustomAugmentations-checkpoint.ipynb
```

This is because [this override](https://github.com/FeriBolour/cotton-phenotyping/blob/3ae6afa0d7e524d7097329dd9427da20190e7c44/.gitattributes#L1) is _not_ recursive:

```gitattributes
Model-Development/* linguist-vendored
```

Change it to:

```gitattributes
Model-Development/** linguist-vendored
```

... and you'll get:

```console
$ github-linguist .
70.27%  153376     Python
29.73%  64884      MATLAB
$
```
